examples/java/src/main/java/org/apache/beam/examples/complete/cdap/SourcePluginWrapper.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.examples.complete.cdap;
+
+import io.cdap.cdap.api.data.batch.InputFormatProvider;
+import org.apache.hadoop.mapreduce.InputFormat;
+
+/**
+ * Class wrapper for a source plugin.
+ */
+public class SourcePluginWrapper<K, V> extends PluginWrapper<InputFormat<K, V>, InputFormatProvider> {
+    public SourcePluginWrapper(InputFormat<K, V> inputFormat, InputFormatProvider inputFormatProvider) {
```

Comment:
Can we add tests that demonstrates creating/usage of these source/sink `PluginWrappers` based on some specific CDAP plugin (e.x. Salesforce) ?

Reply:
Added tests showing examples of `PluginWrapper` and `PluginWrapperBuilder` classes usage